content/developer/smarty/functions.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Functions
+tags:
+- Theming
+- Smarty
+category: developer
+menu:
+  developer:
+    parent: smarty
+aliases:
+- /theming/smarty/conditionals
```

Comment:
This is incorrect. We don't have a "theming" section. It needs to be in "developer"